catalog/core/catalog-core-api-impl/src/main/java/org/codice/alliance/catalog/core/api/impl/types/IsrAttributes.java

```diff
@@ -39,6 +39,24 @@
                 false /* tokenized */,
                 false /* multivalued */,
                 BasicTypes.INTEGER_TYPE));
+        DESCRIPTORS.add(new AttributeDescriptorImpl(SNOW_COVER,
```

Comment:
Could these new attributes starting with `S` be moved to the bottom to maintain the (almost) alphabetical order that currently exists in the file?